core/src/main/java/me/lihq/game/ScenarioBuilder.java

```diff
@@ -119,6 +120,9 @@ Set the first clues in the list to red herrings (the number of red herrings
 
             if (randHidingSpot != null) {
                 room.addClue(clue.setTileCoordinates(randHidingSpot));
+            } else
+            {
+                System.out.println("Failed to place Clue " + clue.getName() + " in room " + room.getName());
```

Comment:
It was yes but it should now never be called as I removed the Secret Room from the room list. But its  good to keep it in for future debugging. I can remove it if everyone wants me to